tests/Test/Pager/Subscriber/Filtration/Doctrine/ORM/QueryTest.php

```diff
@@ -321,10 +488,9 @@ public function shouldFilterSimpleDoctrineQueryWithMultipleProperties(): void
         $dispatcher = new EventDispatcher();
         $dispatcher->addSubscriber(new PaginationSubscriber());
         $dispatcher->addSubscriber(new Filtration());
-        $p = new Paginator($dispatcher);
+        $requestStack = $this->getRequestStack(['filterParam' => 'a.id,a.title', 'filterValue' => '*er']);
```

Comment:
Indentation problem for the whole file :)

Reply:
Can't find indentations problems.... I see 4 spaces in this file